cookbook/demo/agents/creative_studio.py

```diff
@@ -0,0 +1,60 @@
+"""
+Creative Studio - Multimodal Agent with Guardrails and Memory
+"""
+
+from agno.agent import Agent
+from agno.db.postgres import PostgresDb
+from agno.guardrails import PIIDetectionGuardrail, PromptInjectionGuardrail
+from agno.models.openai.chat import OpenAIChat
+from agno.tools.dalle import DalleTools
+from agno.tools.duckduckgo import DuckDuckGoTools
+
+# ============================================================================
+# Database Configuration
+# ============================================================================
+
+db_url = "postgresql+psycopg://ai:ai@localhost:5532/ai"
+db = PostgresDb(db_url, id="creative_studio_db")
+
+# ============================================================================
+# Multimodal Agent with Guardrails
+# ============================================================================
+
+creative_studio = Agent(
+    id="creative-studio",
+    name="Creative Studio",
```

Comment:
```suggestion
    name="Creative Studio",
```

Our recommended syntax here to provide just the name. The id will automatically be `creative-studio` when the name `Creative Studio` is given. So let's do that in these demos!